### ISSUE TITLE
[BUG] Cannot change resolution

### ISSUE THREAD

**Issue by KerrLeada** [pop-os/cosmic-settings#572]
Hi all.

Output from Neofetch in case it matters:
OS: Pop!_OS 24.04
CPU: AMD Ryzen 5 5500U with Radeon Graphics (12) @ 4.056GHz
GPU: AMD ATI 05:00.0 Lucienne

I run the Pop OS 24.04 alpha so I could try out Cosmic. Have found an annoying bug. My display is a TV with a max resolution of 1920x1080. I was playing around with resolutions and did the mistake of selecting 1600x1200. Why is this a mistake? Because now it won't let me change the resolution to anything else. If I for example try and select 1920x1080, it looks like this:
![screenshot-2024-09-07-18-15-45](https://github.com/user-attachments/assets/aff3516e-8ec9-407e-bc2c-837954d99894)
Nothing has changed. I mean that literally as well. The only thing that happens is that the dialog for reverting/keeping changes appears. There is no attempt at changing the resolution itself.

Also in case it matters, 1600x1200 is the wrong aspect ratio, so now everything is stretched. I don't know if this matters to why the bug appared or not, so I'll just mention it. Also not sure if this is the only resolution that has this issue.

I did test to create a new user and see if I could reproduce it. For the new user, the resolution was initially 1920x1080. Steps to reproduce after login:
* Open Settings -> Displays
* Change resolution to 1600x1200
* Select "keep changes"
* Select 1920x1080 again
* Resolution doesn't change

If I try using "cosmic-randr" instead I get this output:
![screenshot-2024-09-07-18-23-39](https://github.com/user-attachments/assets/599778f2-b715-43db-b487-60f3f29c8f5b)
So that doesn't work either (though it's entirely possible I used it incorrectly since it was my first time).

If any more information is needed, please let me know. The reason I put the Pop OS 24.04 alpha on this machine is to try and find bugs like this, so I'm just happy to help.

**Comment by romanstingler** [comment]
don´t panic, now they have removed it completely

**Comment by WatchMkr** [comment]
Has this issue been fixed for you?

**Comment by saraths3** [comment]
Yes i have this issue too, im currently using fedora cosmic spin. but my issue is: ITS ONLY SHOWING MY  NATIVE RESOLUTION.  

  14-inch 2.8K (2880 x 1800) OLED panel. 144hz
  Intel Core 5 210H (8 Cores, up to 4.8 GHz).

I found out that its the issue of cosmic desktop.  

<img width="1574" height="738" alt="Image" src="https://github.com/user-attachments/assets/45078cc0-6177-45a6-9754-b141d382e817" />
